APIs/openEO/federation/backends/collections.md

```diff
@@ -0,0 +1,115 @@
+
+
+## Collection availability
+
+The collections are a key asset of the federation, and users need to know what they can expect from a certain collection.
+The federation relies on openEO collection metadata following the STAC specification to communicate this towards the user. Backend providers
+need to comply with these requirements for their collection:
+
+The 'experimental' flag from https://stac-extensions.github.io/version/v1.1.0/schema.json needs to be set on experimental collections. 
+A collection can be experimental either because there are issues with the actual data or catalog, or because of backend specific issues that make the use of the collection unstable. 
+A collection marked as experimental does not need to comply with further requirements. 
+
+### Requirements for non-experimental collections
+
+The main goal for non-experimental features is to achieve high user satisfaction by providing stability & usability _and_ to comply with federation agreements that make it possible to offer a unified service.
+The list below makes this more concrete, but is not expected to cover all aspects. It is therefore important that providers, who know their implementations and datasets best, properly judge if a specific collection can be
+considered non-experimental.
+
+1. Collections need to indicate the key 'providers' that are responsible for ensuring access to the data and continuity in the case of active missions. The user may depend on the guarantees offered by these
+providers with respect to the properties (timeliness, completeness,...) of a specific collection. 
+The providers with role 'host' and 'producer' are mandatory.
+
+2. The collection description and extents needs to specify known limitations with respect to the original collection. For instance, if only a subset of the full archive is available, this should be indicated. Extents can be rough approximations to avoid requiring very detailed geometry in the metadata.
+
+3. Collections without an end time are assumed to be active missions. By default, 99% of items in these collections should be available within 48 hours after being published by the producer. This gives users a basic guarantee with respect to timeliness of products. 
+4. Collection metadata should be valid STAC metadata and must include all extensions in `stac_extensions`. Tools such as [STAC-validator](https://github.com/stac-utils/stac-validator) can indicate obvious issues.
+5. FAIR principle R1: [(Meta)data are richly described with a plurality of accurate and relevant attributes](https://www.go-fair.org/fair-principles/r1-metadata-richly-described-plurality-accurate-relevant-attributes/)
+6. Collections have to follow harmonization guidelines specified below, if applicable.
+7. Collections naming (id, dimensions, bands) should remain constant. 
+8. Backwards incompatible changes or removal need to be announced with a lead time of 6 months, together with a migration path.
```

Comment:
The need to announce it is already in there?

Reply:
Correct, maybe we need to make it more explicit to announce it to the user and/or explain how to do it. Announcing could mean that they just put a message on their website or documentation but that may not be sufficient?

Reply:
added a section about how to announce things.
We may need to refine it, but would be good to actually create the necessary tags or categories on the forum, so that we can properly link to them.